### PR TITLE
Deprecated Old NetApp Cdot Modules

### PR DESCRIPTION
**Pull request by carchi8py**
##### SUMMARY
Deprecates all NetApp Cdot Modules 

##### ISSUE TYPE

 - Feature Pull Request

##### COMPONENT NAME
na_cdot_*

##### ANSIBLE VERSION

```
ansible 2.7.0.dev0 (ontap-dep 338f8715f5) last updated 2018/08/07 07:52:50 (GMT -700)
  config file = None
  configured module search path = [u'/Users/chrisarchibald/.ansible/plugins/modules', u'/usr/share/ansible/plugins/modules']
  ansible python module location = /private/tmp/ansible/lib/ansible
  executable location = /private/tmp/ansible/bin/ansible
  python version = 2.7.12 (default, Oct 11 2016, 05:20:59) [GCC 4.2.1 Compatible Apple LLVM 8.0.0 (clang-800.0.38)]
```

##### ADDITIONAL INFORMATION

```

```

### CODE REVIEW

**Review by schmots1 — APPROVED** [review]
shipit

**Review by gundalow — CHANGES_REQUESTED** [review]
Can you please add an entry in `docs/docsite/rst/porting_guides/porting_guide_2.7.rst` under `Deprecation notices`

In the format
```
* ``na_cdot_aggregate`` use :ref:`na_ontap_aggregate <na_ontap_aggregate>` instead.
```

**Review by schmots1 — APPROVED** [review]
shipit

**Comment by carchi8py** [comment]
@gundalow i'm not sure what i need to add to get rid of the following error above on porting_guide_2.7

File: docs/docsite/rst/porting_guides/porting_guide_2.7.rst [hunk 1]
```diff
@@ -114,8 +114,16 @@ The following modules no longer exist:
 Deprecation notices
 -------------------
 
-The following modules will be removed in Ansible 2.10. Please update your playbooks accordingly.
-
+The following modules will be removed in Ansible 2.11. Please update your playbooks accordingly.
+
+* ``na_cdot_aggregate`` use :ref:`na_ontap_aggregate <na_ontap_aggregate>` instead.
```
Comment (gundalow) [inline docs/docsite/rst/porting_guides/porting_guide_2.7.rst]: Ah, my mistake, we've changed how the links work, they need `_module` adding.

```
``na_cdot_aggregate`` use :ref:`na_ontap_aggregate <na_ontap_aggregate_module>` instead.
```

**Review by gundalow — APPROVED** [review]

**Comment by gundalow** [comment]
Merged into `devel`

**Pull request merged**